pkg/resourcemonitor/types.go

```diff
@@ -29,7 +29,8 @@ type Args struct {
 	PodResourceSocketPath string
 	SleepInterval         time.Duration
 	Namespace             string
-	KubeletConfigFile     string
+	KubeletConfigURI      string
+	TokenFile             string
```

Comment:
nit'ish: just from the readability point of view the plain name `TokenFile` is somewhat vague ("what token"). Could we rename it to something bit more descriptive, e.g. `APIAuthTokenFile`, `AuthTokenFile` or smth? WDYT?

It might make sense to rename the command line flag, too